pgml-dashboard/templates/cell.html

```diff
@@ -75,7 +75,14 @@
       <div class="notebook-duration">
         <div class="markdown-body">
           <div class="flex flex-row-reverse">
-            <code>TODO</code>
+            <code>Time: <%- 
```

Comment:
Did you consider moving this inside the `sql.html` `<div class="markdown-body">`? I think that might be better to track each statement independently, but I only thought it through because of the warning about an unused variable in the template.

Otherwise, I'd move it to the cell template that uses it.

Reply:
Sorry I totally missed the unused variable error. I switched back to stable from nightly and can see it now.

That was my original plan, but then I saw in the `models.rs` the `Cell` struct already had an `execution_time` attribute and the `cell.html` had a TODO section for it. 

I just pushed a commit that also shows the timing for each sql statement independently. Let me know what you think! Do you want me to remove the rendering of the `Cell execution_time` in the `cell.html`. If so, should I completely remove the `execution_time` attribute for the `Cell` struct and update the migrations to not include that column?